GiniVision/Classes/Core/GiniError.swift

```diff
@@ -36,6 +42,10 @@ public protocol GiniVisionError: Error {}
     /// Unknown error during review.
     case unknown
     
+    public var message: String {
```

Comment:
The same property is present in the Camera error. Can you combine them? It's quite common that an error has a message/description.

Reply:
What's up with this?

Reply:
It is answered above.